ras_party/controllers/queries.py

```diff
@@ -67,6 +67,26 @@ def query_business_respondent_by_respondent_id_and_business_id(business_id, resp
     return response
 
 
+def update_respondent_details(respondent_data, session):
+    """
+    Query to return respondent, respondent_data consists of the following parameters
+    :param respondent_data:
+        respondent_id: id of the respondent
+        first_name:
+        last_name:
+        telephone:
+    :param session
+    :return: updated respondent details
+    """
+
+    logger.debug('Updating respondent details', respondent_id=respondent_data['respondent_id'])
+
+    return session.query(Respondent).filter(Respondent.party_uuid == respondent_data['respondent_id']).update({
```

Comment:
Don't think we even need to return this to be honest?

Reply:
If the query fails this will throw an exception right? If that's the case then we shouldn't have to return anything